### PR TITLE
838 Rename Form to CustomForm::Form and Question to CustomForm::Question

### PR DESCRIPTION
**Pull request by sarvaiyanidhi**
# 🔗 Issue
#838 

# ✍️ Description

Nesting Form to CustomForm::Form
- Changes in Form Model
- Changes in Form Controller
-  Changes in Form Policy
-  Changes in Form View files
- Changes in test cases related to Form

Nesting Question to CustomForm::Question
- Changes in Question Model
- Changes in Question Controller
-  Changes in Question Policy
-  Changes in Question View files
- Changes in test cases related to Question

# 📷 Screenshots/Demos

NA

### CODE REVIEW

File: app/policies/organizations/custom_form/question_policy.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class Organizations::QuestionPolicy < ApplicationPolicy
+class Organizations::CustomForm::QuestionPolicy < ApplicationPolicy
```
Comment (kasugaijin) [inline app/policies/organizations/custom_form/question_policy.rb:3]: Can we nest this as well for consistency?
```
module Organizations
  module CustomForm
    class QuestionPolicy < ...
```

File: app/views/organizations/staff/custom_form/questions/_form.html.erb [hunk 2]
```diff
@@ -26,7 +29,7 @@
       </div>
 
       <%= f.submit t('general.save'), class: 'btn btn-primary' %>
-      <%= link_to t('general.cancel'), staff_forms_path, class: 'btn btn-secondary' %>
+      <%#= link_to t('general.cancel'), staff_forms_path, class: 'btn btn-secondary' %>
```
Comment (kasugaijin) [inline app/views/organizations/staff/custom_form/questions/_form.html.erb:32]: why the `#`?

File: app/views/organizations/staff/custom_form/questions/_form.html.erb [hunk 3]
```diff
@@ -1,6 +1,9 @@
 <div class="row">
   <div class="col-md-8 mx-auto card p-5 bigger">
-    <%= bootstrap_form_with model: [:staff, form, question] do |f| %>
+    <%= bootstrap_form_with(
```
Comment (kasugaijin) [inline app/views/organizations/staff/custom_form/questions/_form.html.erb:3]: Just so I understand, the new namespacing required you to now explicitly provide the URL?

**Comment by kasugaijin** [comment]
@sarvaiyanidhi looking good thank you! Just a couple of comments.

**Comment by sarvaiyanidhi** [comment]
> @sarvaiyanidhi looking good thank you! Just a couple of comments.

Thanks @kasugaijin for feedback. Have addressed reported points and pushed updated code. Let me know if you need any more changes. Thanks

File: app/views/organizations/staff/custom_form/questions/_form.html.erb (hunk 3 above)
Comment (sarvaiyanidhi) [reply app/views/organizations/staff/custom_form/questions/_form.html.erb:3]: Yes, if we don't mention URL it throws issue with route path.

File: app/views/organizations/staff/custom_form/questions/_form.html.erb (hunk 2 above)
Comment (sarvaiyanidhi) [reply app/views/organizations/staff/custom_form/questions/_form.html.erb]: It was commented while making modular changes for code and missed to uncomment later with correct path. Have corrected it and pushed the code.

File: app/policies/organizations/custom_form/question_policy.rb (hunk 1 above)
Comment (sarvaiyanidhi) [reply app/policies/organizations/custom_form/question_policy.rb]: Got missed. Have made the required changes.

**Comment by kasugaijin** [comment]
@sarvaiyanidhi thank you!

**Review by kasugaijin — APPROVED** [review]
LGTM

**Pull request merged**